Patch by Chris Waters fixes installing of several manual pages which
aren't installed by upstream script but are referenced inside other
manual pages.

--- tk8.5-8.5.11.orig/unix/installManPage
+++ tk8.5-8.5.11/unix/installManPage
@@ -91,8 +91,16 @@
 	exit 2 ;;
 esac
 
+Name=`basename $ManPage .$Section`
 SrcDir=`dirname $ManPage`
 
+Specials="FindPhoto FontId MeasureChar"
+for n in $Specials; do
+    if [ "$Name" = "$n" ] ; then
+	Names="$n $Names"
+    fi
+done
+
 ########################################################################
 ### Process Page to Create Target Pages
 ###
